src/lib.rs

```diff
@@ -8,5 +8,4 @@ mod tests {
     fn it_works() {
         assert_eq!(2 + 2, 4);
     }
-
```

Comment:
I know I'm annoying but... Whitespace changes should stay in separate commits :)

Reply:
oopsie, you're right